cql3/statements/modification_statement.cc

```diff
@@ -675,7 +675,11 @@ modification_statement::modification_statement(cf_name name, std::unique_ptr<att
     , _conditions{std::move(conditions)}
     , _if_not_exists{if_not_exists}
     , _if_exists{if_exists}
-{ }
+{
+    if (_attrs->time_to_live) {
+        throw exceptions::invalid_request_exception("TTL attribute is not allowed for deletes");
+    }
+}
```

Comment:
Is this constructor used exclusively for deletes? Its kind of awkward to put this delete-specific code here.

Maybe move to delete_statemet c'tor?

Reply:
aww, vscode tricked me :( I meant to put it in there